packages/alea-frontend/pages/api/acl-utils/acl-common-utils.ts

```diff
@@ -3,14 +3,25 @@ import { NextApiResponse } from 'next';
 import { executeAndEndSet500OnError, getUserIdOrSetError } from '../comment-utils';
 import { CACHE_STORE } from './cache-store';
 
+export async function checkResourcesassociatedOrSet500OnError(
```

Comment:
minor: proper camel casing

```suggestion
export async function checkResourceAssociatedOrSet500OnError(
```

Reply:
fixed